core/data_operations/round_data.py

```diff
@@ -85,6 +78,16 @@ def get_round_data(language):
                     " <a href='https://www.gov.uk/government/publications/community-ownership-fund-prospectus'>"  # noqa
                     " express your interest in the fund</a>."  # noqa
                 ),
+                "support_availability": {
+                    "time": "9yb i 5yp" if language == "cy" else "9am to 5pm",
+                    "days": "Dydd Llun i ddydd Gwener"
+                    if language == "cy"
+                    else "Monday to Friday",
+                    "closed": (
```

Comment:
Don't think we need the `closed` item any more as we aren't displaying this anywhere

Reply:
Aah okay. If you are sure then I can get rid of it? @srh-sloan